solutions/projectstaffing/jgraph/core/src/main/scala/com/microsoft/graphdataconnect/skillsfinder/service/UserService.scala

```diff
@@ -48,41 +49,59 @@ class UserService {
   @Autowired
   var cacheManager: CacheManager = _
 
+  @Value("${anonymous.authentication}")
+  var anonymousUser: Boolean = _
```

Comment:
rename it to isAnonymousAuthEnabled or something like that. also rename app setting to something with 'enabled', like  anonymous.authentication.enabled